.eslintrc

```diff
@@ -353,6 +353,9 @@
                     "virtualization",
                     "vpc",
                     "vpcs",
+                    "webkit",
+                    "webkitdirectory",
+                    "webkitRelativePath",
```

Comment:
Is this actually necessary or was this line maybe just flagged before `webkit` was added?

Reply:
Yeah, both directory, relative, and path should be accepted words. Only webkit should be necessary here